### PR TITLE
fix: panic runtime error: index out of range [7] with length 0

### PR DESCRIPTION
**Pull request by blackbeans**
issue : https://github.com/allegro/bigcache/issues/283

### CODE REVIEW

File: bigcache_test.go [hunk 1]
```diff
@@ -1091,3 +1091,37 @@ func (mc *mockedClock) set(value int64) {
...
+	if nil != err {
+		t.Error(err)
+		t.FailNow()
+	}
+	err = bc.Set("8573", make([]byte, 100))
+	if nil != err {
+		t.Error(err)
+		t.FailNow()
+	}
+	//time.Sleep(5 * time.Second)
```
Comment (janisz) [inline bigcache_test.go:1115]: Please remove commented code

File: bigcache_test.go [hunk 2]
```diff
@@ -1091,3 +1091,37 @@ func (mc *mockedClock) set(value int64) {
...
+
+	err = bc.Set("8573", make([]byte, 450))
+	if nil != err {
+		//t.Error(err)
```
Comment (janisz) [inline bigcache_test.go:1119]: This should be uncommented

**Comment by janisz** [comment]
@cristaloleg @bgalek Can you take a look at this?

File: bigcache_test.go (hunk 1 above)
Comment (blackbeans) [reply bigcache_test.go]: the commented code has been removed

File: bigcache_test.go (hunk 2 above)
Comment (blackbeans) [reply bigcache_test.go]: test case has been modified.

**Review by cristaloleg — APPROVED** [review]
sgtm

**Comment by janisz** [comment]
@blackbeans Looks like tests are failing

**Comment by janisz** [comment]
This PR fixes:

- #283
- #254
- #250
- #234
- #226
- #222

**Comment by blackbeans** [comment]
> @blackbeans Looks like tests are failing

i resubmit . TestCase should use t.Logf to print error ,not use t.Error

**Review by janisz — APPROVED** [review]

**Comment by janisz** [comment]
@cristaloleg PTAL

**Review by cristaloleg — APPROVED** [review]

**Comment by blackbeans** [comment]
Please merge this pull request earlier.

**Comment by janisz** [comment]
@cristaloleg can you merge it? I have no permission to do that's :(

**Comment by flisky** [comment]
Could you please cut a bugfix release? Thanks!

**Comment by janisz** [comment]
Oh, our plugin for releases is not trusted :(
```
toolmantim/release-drafter@v5.2.0 is not allowed to be used in allegro/bigcache. Actions in this workflow must be: within a repository that belongs to your Enterprise account, created by GitHub, verified in the GitHub Marketplace or match the following: JetBrains/*, actions-rs/*, allegro-actions/*, alstr/todo-to-issue-action@v4*, android-actions/*, anothrNick/github-tag-action@1.26.0, appleboy/ssh-action@master, browser-actions/*, coursier/cache-action@v5, crazy-max/ghaction-github-pages@v2.2.0, cypress-io/github-action@*, cypress/*, dflook/terraform-*@v1, docker://artifactory.allegrogroup.com/**, docker://mcr.microsoft.com/mssql-tools:*, docker://openfaas/faas-cli:latest-root, fifsky/ssh-action@*, fregante/daily-version-action@*, golangci/golangci-lint-action@v2, google-github-actions/*, gsactions/commit-message-checker@v1, hashicorp/setup-terraform@v1, jakejarvis/s3-sync-action@*, jitterbit/get-changed-files@v1, jwlawson/actions-setup-cmake@v1.9, mikepenz/release-changelog-builder...
```

**Comment by janisz** [comment]
https://github.com/allegro/bigcache/releases/tag/v3.0.1

**Pull request merged**